web/src/main/webapp/sitemap_general.jsp

```diff
@@ -1,69 +0,0 @@
-<%@ page trimDirectiveWhitespaces="true" %>
-<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
-<%@ page contentType="text/xml; charset=UTF-8" %>
-<%@ page import="java.util.*" %>
-<%@ page import="javax.servlet.http.*" %>
-<%@ page import="org.mskcc.cbio.oncokb.util.PropertiesUtils" %>
-<%@ page import="org.json.simple.parser.JSONParser" %>
-<%@ page import="org.json.simple.JSONArray" %>
-
-<%
-    if (!PropertiesUtils.showSiteMaps()) {
-        response.setStatus(HttpServletResponse.SC_NOT_FOUND);
-    } else {
-        response.setHeader("X-Robots-Tag", "noindex");
-
```

Comment:
By deleting all of these jsp pages, does that mean we will have no legacy webpages at all?

Reply:
Yes, removed all legacy frontend related pages. oncokb-core is really only used for its rest api, so now there's only the swagger page.